docs/loadtest/README.md

```diff
@@ -1,8 +1,7 @@
 # Loadtest
 
-This version of the load tester can test subscription `v1alpha2` only. For `v1alpha1` we have a [legacy branch](https://github.com/kyma-project/eventing-tools/tree/loadtest-subscription-v1alpha1).
-This tool is used to generate continues load on the Eventing components.
-It does that by sending Cloudevents to the Eventing publisher proxy forever and consume them inside Kyma functions.
+This tool is used to generate continuous load on the Eventing components.
+It does that by sending CloudEvents to the Eventing publisher proxy forever and consume them inside Kyma functions.
 Ideally, it should be used when JetStream is used as the active Eventing backend.
 
 ## Eventing infrastructure
```

Comment:
In line 9, it should be **CloudEvents**, not **Cloud Events**

Reply:
Done